From: Juergen Beisert <jbe@kreuzholzen.de>
Subject: S3C24xx/NFC: Remove dead code

Something was to be done here. But I do not remember what. As it works also
without it, remove this dead code.

Signed-off-by: Juergen Beisert <jbe@pengutronix.de>

---
 drivers/mtd/nand/nand_s3c2410.c |    8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

Index: barebox-2011.04.0/drivers/mtd/nand/nand_s3c2410.c
===================================================================
--- barebox-2011.04.0.orig/drivers/mtd/nand/nand_s3c2410.c
+++ barebox-2011.04.0/drivers/mtd/nand/nand_s3c2410.c
@@ -272,8 +272,7 @@ static int s3c2410_nand_correct_data(str
 	 * to see if we have an 0xff,0xff,0xff read ECC and then ignore
 	 * the error, on the assumption that this is an un-eccd page.
 	 */
-	if (read_ecc[0] == 0xff && read_ecc[1] == 0xff && read_ecc[2] == 0xff
-		/* && info->platform->ignore_unset_ecc */)
+	if (read_ecc[0] == 0xff && read_ecc[1] == 0xff && read_ecc[2] == 0xff)
 		return 0;
 
 	/* Can we correct this ECC (ie, one row and column change).
@@ -431,10 +430,7 @@ static int s3c24x0_nand_probe(struct dev
 	mtd->priv = chip;
 
 	/* init the default settings */
-#if 0
-	/* TODO: Will follow later */
-	init_nand_chip_bw8(chip);
-#endif
+
 	/* 50 us command delay time */
 	chip->chip_delay = 50;
 	chip->priv = host;
